lua/mason-registry/semgrep/init.lua

```diff
@@ -0,0 +1,18 @@
+local Pkg = require "mason-core.package"
+local pip3 = require "mason-core.managers.pip3"
+
+return Pkg.new {
+  name = "semgrep",
+  desc = "Semgrep is a fast, open-source, static analysis engine for finding bugs, detecting vulnerabilities in third-party dependencies, and enforcing code standards.",
+  homepage = "https://github.com/returntocorp/semgrep",
+  languages = {
+    Pkg.Lang.Go,
+    Pkg.Lang.Java,
+    Pkg.Lang.Python,
+    Pkg.Lang.Ruby,
+    Pkg.Lang.TypeScript,
+    Pkg.Lang.TypeScriptReact,
```

Comment:
this is new, please let me know if you would like to change

Reply:
Hm I think it might be too specific. These language entries are really just meant to provide the means to do cursory lookups, as an entrypoint for further discovery.

Reply:
I see! What I'm worried about is over-saturating the language list with all kinds of language variants and flavours - it might actually end up negatively impact discoverability if there are too many options.

Reply:
That's fine, I have no qualms either way